qemu/tests/balloon_check.py

```diff
@@ -190,7 +190,7 @@ def balloon_memory(self, new_mem):
         else:
             compare_mem = new_mem
 
-        balloon_timeout = float(self.params.get("balloon_timeout", 240))
+        balloon_timeout = float(self.params.get("balloon_timeout", 360))
```

Comment:
How about double the value to 480s, just in case fail again in high stress env.?

Reply:
It is ok for me ~